packages/salesforcedx-vscode-core/src/constants.ts

```diff
@@ -22,6 +22,9 @@ export const TELEMETRY_GLOBAL_VALUE = 'sfdxTelemetryMessage';
 export const TELEMETRY_OPT_OUT_LINK =
   'https://developer.salesforce.com/tools/vscode/en/faq/telemetry';
 export const TELEMETRY_METADATA_COUNT = 'metadataCount';
+export const BASE_EXTENSION = 'salesforce.salesforcedx-vscode';
+export const EXPANDED_EXTENSION = 'salesforce.salesforcedx-vscode-expanded';
+export const EXT_PACK_STATUS = 'extensionPackStatus';
```

Comment:
It seems most of our telemetry report is snake_case.  Maybe go with something like `sfdx__core_extension_pack_status` to stay with the latest convention?